### PR TITLE
Ixia-c traffic engine deployment on Amazon Web Services with DPDK pull

### PR DESCRIPTION
**Pull request by armdupre**
Includes changes reviewed and approved by @dosarudaniel and @salsous.

### CODE REVIEW

File: public-cloud/aws/ixia-c-dpdk-aws/README.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+## Overview
+This is a public cloud lab where [Ixia-c](https://github.com/open-traffic-generator/ixia-c) has two traffic ports connected within a single subnet of an AWS VPC.
+The environment is deployed using [Terraform](https://www.terraform.io/) and [Cloud-Init](https://cloud-init.io/) is used to configure the application and traffic engines.
+Performance improvements are enabled through [DPDK](https://www.dpdk.org/) support.
+Once the lab is up, a Python script is used to request Ixia-c to generate traffic and report statistics.
+
+![Diagram](./images/diagram.png)
+
+## Prerequisites
```
Comment (bortok) [inline public-cloud/aws/ixia-c-dpdk-aws/README.md:11]: Terraform is required to be installed, need to list this here

File: public-cloud/aws/ixia-c-dpdk-aws/README.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+## Deploy Ixia-c lab
+
+1. Create Terraform deployment.
+
+2. Wait approximately 6 minutes for infrastructure to be deployed.
+
+3. Execute Python test case to generate traffic between AWS instances.
+
+```
+make all
```
Comment (bortok) [inline public-cloud/aws/ixia-c-dpdk-aws/README.md:24]:
```
time terraform apply -auto-approve
╷
│ Error: Failed to load plugin schemas
│
│ Error while loading schemas for plugin components: 5 problems:
│
│ - Failed to obtain provider schema: Could not load the schema for provider registry.terraform.io/hashicorp/aws: failed to instantiate provider
│ "registry.terraform.io/hashicorp/aws" to obtain schema: fork/exec
│ .terraform/providers/registry.terraform.io/hashicorp/aws/5.46.0/linux_amd64/terraform-provider-aws_v5.46.0_x5: invalid argument.
│ - Failed to obtain provider schema: Could not load the schema for provider registry.terraform.io/hashicorp/cloudinit: failed to instantiate provider
│ "registry.terraform.io/hashicorp/cloudinit" to obtain schema: fork/exec
│ .terraform/providers/registry.terraform.io/hashicorp/cloudinit/2.3.4/linux_amd64/terraform-provider-cloudinit_v2.3.4_x5: invalid argument.
│ - Failed to obtain provider schema: Could not load the schema for provider registry.terraform.io/hashicorp/http: failed to instantiate provider
│ "registry.terraform.io/hashicorp/http" to obtain schema: fork/exec
│ .terraform/providers/registry.terraform.io/hashicorp/http/3.4.2/linux_amd64/terraform-provider-http_v3.4.2_x5: invalid argument.
│ - Failed to obtain provider schema: Could not load the schema for provider registry.terraform.io/hashicorp/random: failed to instantiate provider
│ "registry.terraform.io/hashicorp/random" to obtain schema: fork/exec
│ .terraform/providers/registry.terraform.io/hashicorp/random/3.6.1/linux_amd64/terraform-provider-random_v3.6.1_x5: invalid argument.
│ - Failed to obtain provider schema: Could not load the schema for provider registry.terraform.io/hashicorp/time: failed to instantiate provider
│ "registry.terraform.io/hashicorp/time" to obtain schema: fork/exec
│ .terraform/providers/registry.terraform.io/hashicorp/time/0.11.1/linux_amd64/terraform-provider-time_v0.11.1_x5: invalid argument..
╵
```

**Review by bortok — CHANGES_REQUESTED** [review]
Requested changes:
* make it clear in the README to run all the steps in CloudShell
* add "git clone" and "cd" steps before "make all"
* if possible, throw an error if "make all" was run not in CloudShell

File: snappi/data-plane-performance/configs/unidirectional.json [hunk 3]
```diff
@@ -52,15 +52,41 @@
                     "ethernet": {
                         "dst": {
                             "choice": "value",
-                            "value": "00:0c:29:16:b4:61"
+                            "value": "Agent2Eth1MacAddress"
```
Comment (isgmano) [inline snappi/data-plane-performance/configs/unidirectional.json:55]: Isn't this making it specific to AWS only? Same comment for all the changes made in this file for AWS specific requirements. Can this test still run standalone?

File: snappi/data-plane-performance/settings.json [hunk 4]
```diff
@@ -3,11 +3,11 @@
   "http_server": "https://localhost:8443",
   "grpc_server": "localhost:40051",
   "ports": [
-    "TE1_IP:5551",
-    "TE2_IP:5551"
+    "localhost:5551",
```
Comment (isgmano) [inline snappi/data-plane-performance/settings.json:6]: Will this still run as standalone without AWS?

**Review by isgmano — COMMENTED** [review]
Please respond to the comments

**Review by isgmano — APPROVED** [review]

**Pull request merged**